eclair-core/src/main/resources/reference.conf

```diff
@@ -547,6 +547,15 @@ eclair {
     enabled = true // enable automatic purges of expired invoices from the database
     interval = 24 hours // interval between expired invoice purges
   }
+
+  local-reputation {
+    # Reputation decays with the following half life to emphasize recent behavior.
+    half-life = 7 days
+    # HTLCs that stay pending for longer than this get penalized
+    good-htlc-duration = 12 seconds # 95% of successful payments settle in less than 12 seconds, only the slowest 5% will be penalized.
```

Comment:
I guess the stat you're referring to here (95% settling in less than 12 seconds) is based on measurements made on our node? I don't think we should keep this in `reference.conf`?

We could use a better name for this variable too, because a good HTLC settled in *less* than this value, right? So this is not a "good" value. What about `max-htlc-relay-duration`?

Reply:
Done